FlowCal/excel_ui.py

```diff
@@ -736,51 +764,55 @@ def process_samples_table(samples_table,
                                                   num_start=250,
                                                   num_end=100)
             # Remove saturating events in forward/side scatter, and fluorescent
-            # channels to report. The value of a saturating event is taken
-            # automatically from `sample_gated.range`.
-            sample_gated = FlowCal.gate.high_low(sample_gated,
-                                                 sc_channels + report_channels)
+            # channels to report, if the FCS data type is integer. The value of
+            # a saturating event is taken automatically from
+            # `sample_gated.range`.
+            if sample_gated.data_type == 'I':
+                sample_gated = FlowCal.gate.high_low(
+                    sample_gated,
+                    sc_channels + report_channels)
             # Density gating
+            # Axis scaling will be logarithmic if data was acquired with a log
+            # amplifier, and logicle otherwise.
+            sc_amp_type = sample_gated.amplification_type(sc_channels)
             sample_gated, __, gate_contour = FlowCal.gate.density2d(
                 data=sample_gated,
                 channels=sc_channels,
                 gate_fraction=sample_row['Gate Fraction'],
-                xscale='log'
-                       if sample_gated.amplification_type(sc_channels[0])[0]
-                       else 'linear',
-                yscale='log'
-                       if sample_gated.amplification_type(sc_channels[1])[0]
-                       else 'linear',
+                xscale='log' if sc_amp_type[0][0] else 'logicle',
+                yscale='log' if sc_amp_type[1][0] else 'logicle',
```

Comment:
same as mentioned elsewhere: I think amplifier scale and plotting (and gating) scale are orthogonal, so I would still go with logicle, even with a log amplifier.